### PR TITLE
chore(na): apply last defaults

### PR DESCRIPTION
**Pull request by mistic**
Update to last defaults

/CC @evilebottnawi

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,77 +2,80 @@
...
+  "repository": "webpack-contrib/thread-loader",
+  "author": "Tobias Koppers @sokra",
+  "homepage": "https://github.com/webpack-contrib/thread-loader",
+  "bugs": "https://github.com/webpack-contrib/thread-loader/issues",
   "main": "dist/cjs.js",
-  "files": [
-    "dist"
-  ],
+  "engines": {
+    "node": ">= 6.9.0 <7.0.0 || >= 8.9.0"
```
Comment (evilebottnawi) [inline package.json]: Latest `schma-utils` supports only node >= 8.9

File: package.json [hunk 2]
```diff
@@ -2,77 +2,80 @@
...
+    "test:coverage": "cross-env NODE_ENV=test jest --collectCoverageFrom=\"src/**/*.js\" --coverage",
+    "pretest": "npm run lint",
+    "test": "cross-env NODE_ENV=test npm run test:coverage",
+    "defaults": "webpack-defaults"
+  },
+  "files": [
+    "dist"
+  ],
+  "peerDependencies": {
+    "webpack": "^2.0.0 || ^3.0.0 || ^4.0.0"
```
Comment (evilebottnawi) [inline package.json]: I think time to drop webpack@2 and webpack@3

File: package.json (hunk 1 above)
Comment (mistic) [reply package.json]: good catch

File: package.json (hunk 2 above)
Comment (mistic) [reply package.json]: agree

**Comment by evilebottnawi** [comment]
oh, sorry, I done the same in master, did not see your PR when I started working :disappointed:

**Comment by mistic** [comment]
@evilebottnawi that's okay 😄 feel free to close that one

**Pull request closed without merging**